_layouts/default.html

```diff
@@ -10,16 +10,13 @@
     {% endif %}
 
     <meta charset="utf-8">
-    <meta http-equiv=X-UA-Compatible content="IE=edge,chrome=1">
     <meta name=viewport content="width=device-width, initial-scale=1">
-
-    <meta name=description content="{{ site.bio }}">
```

Comment:
Why do you removes the description tag?

Reply:
I don't remember exactly. But this modification was made by recommendation of these tests online.